www/docusaurus.config.js

```diff
@@ -137,6 +137,13 @@ const config = {
         ],
       },
       footer: {
+        logo: {
+          alt: "Firezone Logo",
+          src: "img/logo.svg",
+          href: 'https://www.firezone.dev',
```

Comment:
Consistent quote style:

```suggestion
          href: "https://www.firezone.dev",
```